main.py

```diff
@@ -151,6 +151,15 @@
     slack_app.event("reaction_added")(slack_adapter.reaction_added)
 
     app = FastAPI()
+    app.add_middleware(AuthMiddleware, jwt_secret_key=config.jwt_secret_key, included_routes=[

```

Comment:
Could you clarify what would need to be done to use the login middleware for other endpoints? Do you just add the path here?

Reply:
Yes, just add the path there

Reply:
OK thanks for the info!